### PR TITLE
ARR: Add ethics reviewer unavailability

### PR DESCRIPTION
**Pull request by haroldrubio**

### CODE REVIEW

File: tests/test_arr_venue_v2.py [hunk 1]
```diff
@@ -1478,13 +1504,17 @@ def test_unavailability_process_functions(self, client, openreview_client, helpe
             )
         )
 
+        helpers.await_queue_edit(openreview_client, edit_id=ethics_reviewer_note_edit['id'])
         helpers.await_queue_edit(openreview_client, edit_id=reviewer_note_edit['id'])
         helpers.await_queue_edit(openreview_client, edit_id=ac_note_edit['id'])
         helpers.await_queue_edit(openreview_client, edit_id=sac_note_edit['id'])
 
+        august_ethics_reviewer_edges = {o['id']['tail']: [j['weight'] for j in o['values']] for o in pc_client_v2.get_grouped_edges(invitation=f"{august_venue.get_ethics_reviewers_id()}/-/Custom_Max_Papers", groupby='tail', select='weight')}
```
Comment (melisabok) [inline tests/test_arr_venue_v2.py:1512]: where do you plan to use this edge? is there a matching for ethics reviewers?

Comment (haroldrubio) [reply tests/test_arr_venue_v2.py:1512]: Oh yeah I have to put an edge browser link in the ethics chairs webfield to show the edges

Comment (melisabok) [reply tests/test_arr_venue_v2.py:1512]: can you also push the edge browser link change in the webfield too?

File: openreview/arr/webfield/ethicsChairsWebfield.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+const venueId = domain.id
+const ethicsReviewersId = `${venueId}/${domain.content.ethics_reviewers_name.value}`
+
+const traverse = `${ethicsReviewersId}/-/Assignment`
+const edit = [
+  `${ethicsReviewersId}/-/Assignment`,
+  `${ethicsReviewersId}/-/Invite_Assignment`
+]
+const browse = [
+  `${ethicsReviewersId}/-/Affinity_Score`,
```
Comment (melisabok) [inline openreview/arr/webfield/ethicsChairsWebfield.js:11]: Don't you need to add Custom_Max_Papers here?

File: tests/test_arr_venue_v2.py [hunk 3]
```diff
@@ -1435,10 +1435,21 @@ def test_unavailability_process_functions(self, client, openreview_client, helpe
...
         reviewer_client = openreview.api.OpenReviewClient(username = 'reviewer1@aclrollingreview.com', password=helpers.strong_password)
         ac_client = openreview.api.OpenReviewClient(username = 'ac2@aclrollingreview.com', password=helpers.strong_password)
         sac_client = openreview.api.OpenReviewClient(username = 'sac2@aclrollingreview.com', password=helpers.strong_password)
 
+        ethics_reviewer_note_edit = ethics_reviewer_client.post_note_edit(
+                invitation=f'{august_venue.get_ethics_reviewers_id()}/-/{max_load_name}',
+                signatures=['~EthicsReviewer_ARROne1'],
+                note=openreview.api.Note(
+                    content = {
+                        'maximum_load_this_cycle': { 'value': '4' },
```
Comment (melisabok) [inline tests/test_arr_venue_v2.py:1448]: please use a number here instead of a string.

File: tests/test_arr_venue_v2.py [hunk 4]
```diff
@@ -1485,6 +1500,17 @@ def test_unavailability_process_functions(self, client, openreview_client, helpe
         assert len(august_sac_edges['~SAC_ARRTwo1']) == 1 and set(august_sac_edges['~SAC_ARRTwo1']) == {10}
 
         # Test editing
+        ethics_reviewer_note_edit = ethics_reviewer_client.post_note_edit(
+                invitation=f'{august_venue.get_ethics_reviewers_id()}/-/{max_load_name}',
+                signatures=['~EthicsReviewer_ARROne1'],
+                note=openreview.api.Note(
+                    id = ethics_reviewer_note_edit['note']['id'],
+                    content = {
+                        'maximum_load_this_cycle': { 'value': '5' },
```
Comment (melisabok) [inline tests/test_arr_venue_v2.py:1509]: same here, use 5 instead of '5'

**Review by melisabok — APPROVED** [review]

**Pull request merged**